src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/default-metrics-and-metric-attributes.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Default metric labels and metric attributes'
```

Comment:
the page name and file name differs. One specify "Default metric labels and.." the second one "Default metrics and..."